specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-11-01/imds.json

```diff
@@ -581,6 +591,10 @@
         "virtualTpmEnabled": {
           "type": "string",
           "description": "Identifies if the virtual Trusted Platform Module (TPM) is enabled on the VM"
+        },
+        "encryptionAtHost": {
+          "type": "string",
+          "description": "Identifies if encryption at host is enabled on the VM"
         }
```

Comment:
Using a string for boolean is confusing, but given that you've already having it in GA and you having same for a couple of other properties, I guess there is nothing I can do here.